modules/netflow/configuration/logstash/netflow.conf.erb

```diff
@@ -182,6 +182,19 @@ filter {
         }
 
         # Populate bytes transferred in the flow.
+
+        if [netflow][fwd_flow_delta_bytes] {
+            mutate {
+                id => "netflow-v9-normalize-bytes-from-fwd_flow_bytes"
+                rename => { "[netflow][fwd_flow_delta_bytes]" => "[netflow][bytes]" }
+            }
+        }
+        if [netflow][rev_flow_delta_bytes] {
```

Comment:
should be `else if ` -- we don't want to overwrite the value unnecessarily if both are present.